tests/poetry_scripts.py

```diff
@@ -0,0 +1,19 @@
+import subprocess
+
+def run_unit_tests():   
+    """
+    Run all unittests. Equivalent to:
+    `poetry run python -u -m unittest discover tests/unit`
+    """
+    subprocess.run(
```

Comment:
Is there an easier way to do this without subprocess?

Reply:
Looks like we could use a plugin: https://github.com/nat-n/poethepoet#configuring-the-plugin (there might be other options as well, this was just the one linked in the PR that you linked).
I think that this would be really useful, so a plugin would be worth it.

Reply:
I installed it as a dev dependency and seems to work just by calling `poe test`